Sources/MySQL/Connection/MySQLData.swift

```diff
@@ -583,40 +583,33 @@ extension Date {
             _comps.currentValue = new
             comps = new
         }
-        /// For some reason comps.nanosecond is `nil` on linux :(
-        let nanosecond: Int
-        #if os(macOS)
-        nanosecond = numericCast(time.microsecond) * 1_000
-        #else
-        nanosecond = 0
-        #endif
-        
+
         comps.value.year = numericCast(time.year)
         comps.value.month = numericCast(time.month)
         comps.value.day = numericCast(time.day)
         comps.value.hour = numericCast(time.hour)
         comps.value.minute = numericCast(time.minute)
         comps.value.second = numericCast(time.second)
         comps.value.nanosecond = numericCast(time.microsecond) * 1_000
```

Comment:
Should this line be removed too?

Reply:
Meh space for clarity is never a bad thing!